packages/regl-worldview/src/stories/GLText.stories.js

```diff
@@ -41,6 +42,50 @@ function textMarkers({
   });
 }
 
+// Originally from: https://css-tricks.com/converting-color-spaces-in-javascript/
```

Comment:
Maybe we can add [tinycolor2](https://www.npmjs.com/package/tinycolor2)
  as dependency since we use in webviz-core anyhow?

Reply:
oh cool! did not know we used this lib -- will refactor.